doc/design/paths-endpoint-webdav.md

```diff
@@ -0,0 +1,21 @@
+# Discussion on metadata needed for dandidav
+
+Submitted as a PR to avoid spagetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
+
+
+@jwodder listed needed properties and metadata fields:
+
+* Asset properties used by dandidav:
+    * `asset_id`
+    * `blob_id`
+    * `zarr_id`
+    * `path`
+    * `size`
+    * `created`
+    * `modified`
+        * Note: It might be more accurate to use the `blobDateModified` metadata field for this instead; cf. dandi/dandidav#17
+
+* Asset metadata used by dandidav:
+    * `encodingFormat` (blobs only)
+    * `contentUrl` (API download URL for blobs, S3 URL for Zarrs)
+    * `digest["dandi:dandi-etag"]` (blobs only)
```

Comment:
what this one for?

Reply:
in principle could be any other checksum I guess , right?

note: I thought that besides ETag functionality a commonly known checksum could then be used by receiving end to validate the download if that would be the target purpose for such a listing. 

for versioned zarrs we would need `"dandi:dandi-zarr-checksum"` to use in conjunction with the `zarr_id`.

Reply:
Well, the `dandi:dandi-etag` is the same as the ETag header reported by S3 when downloading a blob asset.  In contrast, there's no single file that has a `dandi:dandi-zarr-checksum` as an ETag.

Reply:
I meant that we can use `dandi:dandi-zarr-checksum ` as a value for  ETag on zarr collections (folders).

Reply:
I'm not sure if it makes sense to apply etags to collections, as `GET`ing a collection won't give you a response with that etag.  (Cf. https://github.com/dandi/dandidav/issues/26)